roles/pulp_common/README.md

```diff
@@ -134,7 +134,21 @@ RPM packages instead if this variable is set. Other distro packaging formats may
   (RPM) "packages".
   Defaults to "pip".
 
-If it is set to "packages", the following variables are used, or behave *differently* from above:
+If it is set to "packages", the installer is in packges mode, which has the following **limitations**:
+
+* Only CentOS/RHEL 7 and CentOS/RHEL 8 are supported.
+* Not all plugins are available from the default repo. To determine which plugins are available,
+  follow [this link](https://yum.theforeman.org/pulpcore/), browse to the repo for your
+  pulp_version and distribution, and search for "python3-pulp-".
```

Comment:
That's not strictly true. In recent versions the prefix is `tfm-pulpcore-python3-pulp-` on EL7 and `python38-pulp-` on EL8.

Reply:
Yeah, I forgot about that.

How about we tell them to search for "pulp-"? It will list the CLI and a few other non-plugin-packages packages, but technically listing `pulp-cli` under `pulp_install_plugins`, is a method for installing them.

Reply:
I made this change.